src/Test.UnitTests.Sarif/TestData/SarifCurrentToVersionOneVisitor/Inputs/OneRunWithInvocationAndNotifications.sarif

```diff
@@ -31,7 +31,7 @@
               },
               "level": "error",
               "threadId": 52,
-              "time": "2016-07-16T14:18:43.119Z",
```

Comment:
>time" [](start = 15, length = 5)

we noted this bug in a recent PR. we incorrectly assumed that this was a transformer bug. Actually, its not! This was an incorrect schema issue in the test.

we have code for time -> timeUtc transformation. However, that transformation applies prior to "2018-10-10" release. So, i have fixed this test to be a valid "2018-10-10" schema, where the field is called timeUtc, not time.

But to cross check that our transformation correctly works, i have added another test case (with an older schema version).